tests/test_router_bgp.rb

```diff
@@ -20,17 +20,38 @@
 require_relative 'ciscotest'
 require_relative '../lib/cisco_node_utils/bgp'
 
+XR_NOT_SUPPORTED     = 'Not supported in IOS XR'
```

Comment:
I think the skip count is high because of what you pointed out before - I am skipping whole test cases. But I'll do an assessment and get back to you once I've changed those tcs.